roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -24,7 +24,7 @@ server {
   {% if item.value.multisite.enabled | default(false) -%}
     {% if item.value.multisite.subdomains | default(false) -%}
       rewrite ^/(wp-.*.php)$ /wp/$1 last;
-      rewrite ^/(wp-(content|admin|includes)/.*) /wp/$1 last;
+      rewrite ^/(wp-(content|admin|includes).*) /wp/$1 last;
```

Comment:
this seems like a core bug since xmlrpc breaks when WP is installed in a subdirectory. i'm not sure this is something that we should add to trellis, but it's something we could at least cover in the wiki/docs.